### PR TITLE
remoteBackup: fix invalid config file breaks Artifactory instance

### PR DESCRIPTION
**Pull request by diepfote**

### CODE REVIEW

**Comment by florianbegusch** [comment]
I have read the CLA Document and I hereby sign the CLA

File: storage/remoteBackup/remoteBackup.groovy [hunk 1]
```diff
@@ -51,7 +51,13 @@ storage {
   afterCreate { item ->
     def etcdir = ctx.artifactoryHome.etcDir
     def cfgfile = new File(etcdir, REMOTE_BACKUP)
-    def cfg = new JsonSlurper().parse(cfgfile)
+
+    // Ensure invalid json config does not prevent Artifactory
+    // from serving and receiving artifacts.
+    if (!cfgfile.exists()) return
```
Comment (AlexanderDenk-EE) [inline storage/remoteBackup/remoteBackup.groovy:57]: A log message for this event would be nice.

**Comment by AlexanderDenk-EE** [comment]
@DarthFennec Can you please provide a review?

**Comment by diepfote** [comment]
@yashprit-jfrog we integrated this internally. Feel free to close

**Comment by yashprit-jfrog** [comment]
Closing based on above comment

**Pull request closed without merging**